### PR TITLE
Revert langdef.md changes

### PR DESCRIPTION
**Pull request by ayas**
Revert the changes back to the one that override them(from #16 to #12).

### CODE REVIEW

File: doc/langdef.md [hunk 1]
```diff
@@ -950,7 +1269,7 @@ descriptions need to be updated in the code.
       google.protobuf.Timestamp.(string) -> int
     </td>
     <td>
-      get day of month from the date with timezone, one-based indexing
+      get day of month from the date with timezone, zero-based indexing
```
Comment (JimLarson) [inline doc/langdef.md]: This and the below timezone-related changes were not part of pull #11, so I think they're part of a third_party change that hadn't been synced to GitHub yet, so don't revert these changes.

Comment (JimLarson) [inline doc/langdef.md]: This and the below timezone-related changes were not part of pull #11, so I think they're part of a third_party change that hadn't been synced to GitHub yet, so don't revert these changes.

File: doc/langdef.md [hunk 2]
```diff
@@ -1072,7 +1391,7 @@ descriptions need to be updated in the code.
       google.protobuf.Timestamp.(string) -> int
     </td>
     <td>
-      get milliseconds from the date with timezone, 0-999
+      get milliseconds from the date in UTC, 0-999
```
Comment (JimLarson) [inline doc/langdef.md]: Here

File: doc/langdef.md [hunk 3]
```diff
@@ -1383,46 +1702,6 @@ descriptions need to be updated in the code.
       type conversion
     </td>
   </tr>
-  <tr>
-    <th rowspan="1">
```
Comment (JimLarson) [inline doc/langdef.md]: Here

File: doc/langdef.md [hunk 4]
```diff
@@ -1383,46 +1702,6 @@ descriptions need to be updated in the code.
...
-    <td>
-      (string) -> google.protobuf.Duration
-    </td>
-    <td>
-      type conversion, duration should be end with "s", which stands for seconds
-    </td>
-  </tr>
 </table>
 
-Timezone referenced to ftp://ftp.iana.org/tz/, data version: 2018c.
```
Comment (JimLarson) [inline doc/langdef.md]: ...and here.

**Comment by JimLarson** [comment]
Spoke with Selcuk, who is uncomfortable with dealing with the timezone-related changes and prefers to merge them in third_party.

File: doc/langdef.md (hunk 1 above)
Comment (ayas) [reply doc/langdef.md]: I suggest we first revert this file to its original, sync these changes to third_party/ and then sync the additional changes from third_party to here.
